README.md

```diff
@@ -198,7 +198,7 @@ When using Coverage.jl locally, over time a lot of `.cov` files can accumulate.
 
        ```yml
        after_test:
-       - C:\projects\julia\bin\julia -e "using Pkg; Pkg.add(\"Coverage\"); using Coverage; Coveralls.submit(process_folder())"
+       - C:\julia\bin\julia -e "using Pkg; Pkg.add(\"Coverage\"); using Coverage; Coveralls.submit(process_folder())"
```

Comment:
I don't know anything about Julia. What is the current (or past) default install directory ?

Reply:
Oh wait, now I realize this is about AppVeyorand in fact a duplicate of PR #329